### PR TITLE
Refactor tests for recent changes

### PR DESCRIPTION
**Pull request by rexagod**
Fixes tests for changes introduced in:
* https://github.com/red-hat-storage/mcg-osd-deployer/pull/38
* https://github.com/red-hat-storage/mcg-osd-deployer/pull/16

Pulls in changes from (in addition to its own changes):
* https://github.com/red-hat-storage/mcg-osd-deployer/pull/44
* https://github.com/red-hat-storage/mcg-osd-deployer/pull/41

Signed-off-by: Pranshu Srivastava <rexagod@gmail.com>

### CODE REVIEW

File: tests/managedmcg_controller_test.go [hunk 1]
```diff
@@ -82,6 +86,26 @@ var _ = Describe("ManagedMCGReconciler Reconcile", func() {
...
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "mcg-ms-console",
+				Namespace: namespace,
+			},
+		}
+		consolePluginFake := consolev1alpha1.ConsolePlugin{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "mcg-ms-console",
+			},
+			// Spec: consolev1alpha1.ConsolePluginSpec{
```
Comment (naveenpaul1) [inline tests/managedmcg_controller_test.go:99]: please remove commented code

**Review by naveenpaul1 — APPROVED** [review]
LGTM

File: tests/managedmcg_controller_test.go (hunk 1 above)
Comment (rexagod) [reply tests/managedmcg_controller_test.go]: It looks like this was introduced here: https://github.com/red-hat-storage/mcg-osd-deployer/pull/44/files#diff-95527ea8bbfbfa57170f8b6d8cf862f19fa4037cc7ea51e88e003bb982e3febeR99-R107.

Removed.

**Comment by rexagod** [comment]
<details>
<summary>Verified.</summary>

Screenshot from 2022-06-08 15-01-13
Screenshot from 2022-06-08 15-01-04

</details>

Please LGTM.

**Pull request merged**